PennMobile/LaundryPrototype/Views/LaundryRowView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  LaundryRowView.swift
+//  PennMobile
+//
+//  Created by Nathan Aronson on 10/20/25.
+//  Copyright © 2025 PennLabs. All rights reserved.
+//
+
+
+import SwiftUI
+
+struct LaundryRowView: View {
+    
+    let hall: LaundryHallId
+    let isSelected: Bool
+    let canSelect: Bool
+    let toggle: () -> Void
```

Comment:
Could we rely on an `@Binding` for the selection state and `@Environment(\.isEnabled)` for `canSelect`?

You can specify your own `Binding` get/set logic on the caller's end if this doesn't map to a single `@State`.